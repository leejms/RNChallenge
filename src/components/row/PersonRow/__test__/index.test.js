import 'react-native'
import React from 'react'
import Component from '../index'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(<Component
      avatar='https://www.planwallpaper.com/static/images/9-credit-1.jpg'
      title='TEST'
      subtitle='test'
      following={true}
      follow={() => console.log(item.id)}/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})