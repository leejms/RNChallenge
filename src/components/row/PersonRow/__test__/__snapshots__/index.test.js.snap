// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "borderBottomColor": "#F5F5F5",
      "borderBottomWidth": 1,
      "flex": 1,
      "flexDirection": "row",
      "height": 90,
      "padding": 15,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": 60,
        "justifyContent": "center",
        "width": 60,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://www.planwallpaper.com/static/images/9-credit-1.jpg",
        }
      }
      style={
        Object {
          "borderRadius": 30,
          "height": 60,
          "width": 60,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flex": 4,
        "height": 90,
        "justifyContent": "flex-start",
        "paddingLeft": 20,
        "paddingTop": 5,
      }
    }
  >
    <View
      style={Object {}}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#000000",
            "fontFamily": "Roboto-Regular",
            "fontSize": 16,
            "fontWeight": "bold",
          }
        }
      >
        TEST
      </Text>
    </View>
    <View
      style={Object {}}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#a5a5a5",
            "fontFamily": "Roboto-Regular",
            "fontSize": 14,
          }
        }
      >
        test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          unfollow
        </Text>
      </View>
    </View>
  </View>
</View>
`;
